components/list/list-item-checkbox-mixin.js

```diff
@@ -76,7 +71,12 @@ export const ListItemCheckboxMixin = superclass => class extends superclass {
 
 	_renderCheckboxAction(inner) {
 		return this.selectable ? html`
-			<div @click="${this._handleCheckboxActionClick}" class="d2l-checkbox-action" ?disabled="${this.disabled}">${inner}</div>
+			<label
+				class="d2l-checkbox-action"
+				?disabled="${this.disabled}"
+				for="${this._checkboxId}">
+				${inner}
+			</label>
```

Comment:
Changing this to a `label` makes the click work natively in browsers, so need for a handler.